pkg/storage/mongo/mongo.go

```diff
@@ -0,0 +1,41 @@
+package mongo
+
+import (
+	"github.com/globalsign/mgo"
+)
+
+type MongoModuleStore struct {
+	s   *mgo.Session
+	d   string // database
+	c   string // collection
+	url string
+}
+
+// NewMongoStorage  returns an unconnected Mongo Module Storage
+// that satisfies the Storage interface.  You must call
+// Connect() on the returned store before using it.
```

Comment:
we should indicate that this is an implementation of `StorageConnector`

I can do this in a follow-up 😄